speechbrain/nnet/RNN.py

```diff
@@ -1199,13 +1196,16 @@ def __init__(
         # Setting the activation function
         if nonlinearity == "tanh":
             self.act = torch.nn.Tanh()
-        elif nonlinearity == "sin":
-            self.act = torch.sin
         elif nonlinearity == "leaky_relu":
             self.act = torch.nn.LeakyReLU()
         else:
             self.act = torch.nn.ReLU()
 
+        from speechbrain.nnet.ligru.cuda_ligru_cell import _ligru_cell_jit
```

Comment:
Hum, enforcing Jit is not following the philosophy of SB. In practice, using jit or not is defined in the YAML, see for instance [this recipe](https://github.com/speechbrain/speechbrain/blob/develop/recipes/TIMIT/ASR/seq2seq/hparams/train.yaml). Here, it should be the same. If the encoder is passed to the (jit_module_keys) paremeter in the yaml THEN your stuff should be jitted, otherwise, nope. Maybe it's already the case and you don't have to add this wrapper.